ezGroceriesShoppingList/src/main/java/com/ezgroceries/shoppinglist/web/WebController.java

```diff
@@ -0,0 +1,68 @@
+package com.ezgroceries.shoppinglist.web;
+
+import com.ezgroceries.shoppinglist.entity.Cocktail;
+import com.ezgroceries.shoppinglist.entity.ShoppingList;
+import com.ezgroceries.shoppinglist.service.CocktailService;
+import com.ezgroceries.shoppinglist.service.ShoppingListService;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@Slf4j
+public class WebController {
```

Comment:
It's best to apply the separation of control principle. In this case we would have 2 controllers, one for cocktails, one for shopping carts. For now this is okay and we will leave it, but for the next PR, you should separate them